features/input-event.yml

```
@@ -1,4 +1,4 @@
-name: input (event)
+name: Input event
```

This came about in https://github.com/web-platform-dx/web-features/pull/1026 and https://github.com/web-platform-dx/web-features/pull/1492. Let's leave this for now, then do a full audit of event features when we've mapped everything.

```suggestion
name: input (event)
```